### PR TITLE
Rename OwnedValidators.activeValidators in favor of "enabled"

### PR DESCRIPTION
**Pull request by zilm13**
## PR Description
We have  `OwnedValidators.getActiveValidators()` which spreads to several other classes and it's often confused with [ValidatorStatus](https://github.com/ConsenSys/teku/blob/f31b239f7788f07d8e9c2b6e7544c5e16b01a259/data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/beacon/ValidatorStatus.java) while it means only whether validator is managed by node or not. Renamed to avoid confusion in the future

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Comment by tbenr** [comment]
To me seems like this is just `getValidators()`. `enabled` or everything else would be justified if we could get validators that are "not" enabled. But to me seems like `OwnedValidators` can just have or not have a validator.

**Comment by StefanBratanov** [comment]
I think i prefer `owned` instead of `enabled` so method will be `OwnedValidators.getValidators()` and then `OwnedKeysManager` and in comments etc.

**Comment by zilm13** [comment]
@StefanBratanov yeah, now looks better to me. Tried to move this way.

**Review by rolfyone — APPROVED** [review]
Good improvement!

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/OwnedKeyManager.java [hunk 1]
```diff
@@ -44,41 +44,36 @@
...
-  public ActiveKeyManager(
+  public OwnedKeyManager(
       final ValidatorLoader validatorLoader, final ValidatorTimingChannel validatorTimingChannel) {
     this.validatorLoader = validatorLoader;
     this.validatorTimingChannel = validatorTimingChannel;
   }
 
-  /**
-   * Get a listing of active validator keys
-   *
```
Comment (StefanBratanov) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/OwnedKeyManager.java:62]: Just this comment needs changing I think

**Review by StefanBratanov — APPROVED** [review]

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/OwnedKeyManager.java (hunk 1 above)
Comment (zilm13) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/OwnedKeyManager.java:62]: We agreed that just removing it is ok

**Pull request merged**